src/Sylius/Component/Core/Repository/PromotionRepositoryInterface.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Repository;
+
+use Sylius\Component\Core\Model\ChannelInterface;
+use Sylius\Component\Promotion\Repository\PromotionRepositoryInterface as BasePromotionInterface;
```

Comment:
I don't alias it ``BasePromotionRepositoryInterface`` because I haven't noticed this bug :smile: :+1: